scripts/populate_sample_contributor_data.py

```diff
@@ -101,6 +101,8 @@ def populate_data(self) -> None:
         self._sign_in(SUPER_ADMIN_EMAIL)
         self.csrf_token = self._get_csrf_token()
         self._assign_admin_roles(SUPER_ADMIN_ROLES, SUPER_ADMIN_USERNAME)
+        self._add_review_translation_rights(SUPER_ADMIN_USERNAME)
```

Comment:
Pass langauge_code as param! This will make it easier to understand what role is assigned to the user just by reading this section. (OR change the method name to `_add_review_akan_translation_rights`)